src/parsing.rs

```diff
@@ -23,7 +23,7 @@ pub type ParseResult<T> = Result<T, ParseError>;
 impl ::std::fmt::Display for ParseError {
     fn fmt(
         &self,
-        fmt: &mut ::std::fmt::Formatter<'_>,
```

Comment:
Changes to this file seem unnecessary. Let's undo those.

Reply:
Done.